compiler/x/codegen/X86Ops.hpp

```diff
@@ -1418,7 +1528,7 @@ class TR_X86OpCode
          }
       return len + getOpCodeLength();
       }
-   uint8_t* binary(uint8_t* cur, uint8_t rex = 0)
+   uint8_t* deprecated_binary(uint8_t* cur, uint8_t rex)
```

Comment:
deprecated functions need a comment about why they were deprecated and what their replacements is/are.